modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -310,6 +310,14 @@ CV__DNN_INLINE_NS_BEGIN
         static Ptr<GatherLayer> create(const LayerParams& params);
     };
 
+    /** @brief GatherElements layer
+     */
+    class CV_EXPORTS GatherElementsLayer : public Layer
+    {
+    public:
+        static Ptr<GatherElementsLayer> create(const LayerParams& params);
+    };
+
```

Comment:
Thanks for contribution! Please add a brief description about the difference between `Gather` and `GatherElements`

Reply:
Done!